docs/debugdrawer-bridge.md

```diff
@@ -0,0 +1,105 @@
+---
+layout: page
+title: DebugDrawer Bridge
+---
+
+DebugDrawer Bridge
+=============
+
+This component uses DriveChain's application hooks to automatically initialize the [DebugDrawer].
+
+Installation
+------------
+
+Add the dependency to your `build.gradle` file
+
+```groovy
+    debugCompile "com.github.InkApplications.DriveChain:debugdrawer-bridge:1.+"
+    releaseCompile "com.github.InkApplications.DriveChain:debugdrawer-bridge-noop:1.+"
+```
+
+And any additional modules you wish to include
+
+```groovy
+    implementation "com.github.InkApplications.DriveChain:debugdrawer-location-bridge:$driveChainVersion"
+    implementation "com.github.InkApplications.DriveChain:debugdrawer-logs-bridge:$driveChainVersion"
+    implementation "com.github.InkApplications.DriveChain:debugdrawer-okhttp-bridge:$driveChainVersion"
+```
+ 
+Add the module to your application component:
+
+```kotlin
+    @Singleton
+    @Component(modules = arrayOf(
+        DebugDrawerBridgeModule::class
+    ))
+    interface ApplicationComponent {
+      // ...
+    }
+```
+
+Usage
+-----
+
+Simply inject this into an Activity for it to appear in debug builds. Provide the `DebugDrawer` from the module scoped with an activity.
+
+```kotlin
+    @Module(includes = [
+        DebugDrawerBridgeModule::class,
+        DebugDrawerOkHttpBridgeModule::class,
+        DebugDrawerLogsBridgeModule::class
+    ])
+    class MainModule {
+        @Provides @ActivityScope fun debugDrawer(activity: MainActivity, debugModules: Set<@JvmSuppressWildcards DebugModule>) : DebugDrawer {
```

Comment:
I've been trying to hide the `@JvmSuppressWildcards` by wrapping them in another object, typealias, or someday an inline class.
I think a typealias would work here.

Reply:
Also I don't get why we have to build this ourself. Can the bridge provide this for us?